Issue Template Correct?

Replace the values below using the output from `npm list storj`.

```
root@flooder1:~# npm list storj
/root
└── (empty)

npm ERR! code 1
```

Should we instead ask them to simply do ```storj --version```?

--- a/ISSUE_TEMPLATE.md
+++ b/ISSUE_TEMPLATE.md
@@ -1,6 +1,6 @@
 ### Package Versions
 
-Replace the values below using the output from `npm list storj`.
+Replace the values below using the output from `npm list storj`. Use ```npm list -g storj``` if installed globally.
 
 ```
 storj-bridge@0.6.5 /home/gordon/Code/storj-bridge

